File: src/main/java/dto/ResponseResult.java
package dto;

public class ResponseResult<T> {
    private int response_code;
    private T results;
    public ResponseResult() {
    }

    public ResponseResult(int response_code, T results) {
        this.response_code = response_code;
        this.results = results;
    }

    public int getResponse_code() {
        return response_code;
    }

    public void setResponse_code(int response_code) {
        this.response_code = response_code;
    }

    public T getResults() {
        return results;
    }

    public void setResults(T results) {
        this.results = results;
    }
}

File: src/main/webapp/scripts/responseResult.js
class ResponseResult {
    constructor(response_code, results) {
        this.response_code = response_code;
        this.results = results;
    }

    get response_code() {
        return this._response_code;
    }

    set response_code(value) {
        this._response_code = value;
    }

    get results() {
        return this._results;
    }

    set results(value) {
        this._results = value;
    }
}

export { ResponseResult };
